tests/har.spec.ts

```diff
@@ -23,7 +23,7 @@ import type { BrowserContext, BrowserContextOptions } from 'playwright-core';
 import type { AddressInfo } from 'net';
 import type { Log } from '../packages/playwright-core/src/server/supplements/har/har';
 
-async function pageWithHar(contextFactory: (options?: BrowserContextOptions) => Promise<BrowserContext>, testInfo: any, outputPath: string = 'test.har') {
+export async function pageWithHar(contextFactory: (options?: BrowserContextOptions) => Promise<BrowserContext>, testInfo: any, outputPath: string = 'test.har') {
```

Comment:
If we export this, let's make it a fixture? Importing specs from other specs is error-prone.

Reply:
Sounds good! Re-located as a fixture. Depending on what tests I end up with, I might put the SW HAR test alongside the other HAR bits. (During dev, it's just nice to have all them in one file 😄 )